src/alert-converter/alert-converter.service.ts

```diff
@@ -24,19 +36,110 @@ export class AlertConverterService {
             if (alertStatus == "firing") {
 
                 const date = new Date(alert.startsAt);
-                const log: LogMessageFormat = {
-                    type: LogType.CPU,
-                    time: date.getTime(),
-                    source: null, //TODO: Define source
-                    detector: 'Prometheus',
-                    message: alert.annotations.description,
-                    data: {
-                        cpuUtilization: 0,
-                    },
-                };
+                let log: LogMessageFormat;
+                
+                //check for correct LogType
+                switch (this.getLogType(alert.labels.alertname)) {
+                    case LogType.CPU:
+                        log = {
+                            type: LogType.CPU,
+                            time: date.getTime(),
+                            source: this.windowsExporterUrl,
+                            detector: this.prometheusUrl,
+                            message: alert.annotations.description,
+                            data: {
+                                cpuUtilization: this.getCPULoadOfString(alert.annotations.description),
+                            },
+                        };
+                        break;
+
+                    case LogType.TIMEOUT:
+                        log = {
+                            type: LogType.TIMEOUT,
+                            time: date.getTime(),
+                            source: this.windowsExporterUrl,
+                            detector: this.prometheusUrl,
+                            message: alert.annotations.description,
+                            data: {
+                                timeoutDuration: 0 //Can be specified when an timeout rule exists.
+                            },
+                        };
+                        break;
+
+                    case LogType.ERROR:
+                        log = {
+                            type: LogType.ERROR,
+                            time: date.getTime(),
+                            source: this.windowsExporterUrl,
+                            detector: this.prometheusUrl,
+                            message: alert.annotations.description,
+                            data: null,//Can be specified when an timeout rule exists.
```

Comment:
```suggestion
                            data: null,//Can be specified when a timeout rule exists.
```
deweiteren... wieso hängen die daten eines error logs von einer timeout rule ab?